indykite/common_schema.go

```diff
@@ -164,7 +164,7 @@ func exactNameFilterSchema() *schema.Schema {
 func apiPermissionsSchema() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeList,
-		Optional: true,
+		Required: true,
 		Elem: &schema.Schema{
 			Type:         schema.TypeString,
 			ValidateFunc: validation.StringLenBetween(1, 64),
```

Comment:
Suggestion: Just change to validation, so you validate there is value from the list. 

Example like this

```go
validation.StringInSlice([]string{
	"europe-west1", "us-east1",
}
```